ksqldb-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -618,6 +619,27 @@ private void printSourceDescription(final SourceDescription source) {
         "Statistics of the local KSQL server interaction with the Kafka topic "
             + source.getTopic()
     ));
+    Optional<SourceConsumerOffsets> consumerGroupOffsetsOptional = source.getConsumerGroupOffsets();
```

Comment:
nit: our checkstyle should enforce some things like local variables being `final` whenever possible, you can run `mvn checkstyle:checkstyle` to make sure that your code passes all checkstyle checks